Cargo.toml

```diff
@@ -11,14 +11,16 @@ crate-type = ["cdylib", "rlib"]
 [features]
 default = ["export"]
 export = []
-external = ["stellar-contract-sdk/testutils", "dep:ed25519-dalek", "dep:num-bigint", "dep:sha2", "dep:stellar-xdr"]
+external = ["stellar-contract-sdk/testutils", "dep:ed25519-dalek", "dep:num-bigint", "dep:sha2", "stellar-xdr/num-bigint"]
```

Comment:
Why'd you remove `dep:stellar-xdr`? Is that implied by `stellar-xdr/num-bigint`?

Reply:
You're correct. It is implied, and in fact the two are incompatible. Since the latter enables the stellar-xdr crate, it makes the presence of the dep: version an error since once you refer to a dep with dep: you can't refer to it any other way.

Reply:
Actually, I just pushed 6ca1bbce31c7687f2dda8630adcc27230e4a9b35 which I think is preferable as it preserves the behavior of making the `stellar-xdr` feature not a feature.

Reply:
Interesting, thanks for the explanation!